Traceback (most recent call last):
  File "/home/ubuntu/CatDB/Experiments/catdb-results/dataset_5_rnc/gpt-4/dataset_5_rnc-SCHEMA-Random-0-SHOT-gpt-4.py", line 20, in <module>
    train_data = train_data.replace([np.inf, -np.inf], np.nan)
                                     ^^
NameError: name 'np' is not defined
